### ISSUE TITLE
Accessibility: MAS 42A: High Contrast:  The download link arrows are missing in high contrast black

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#12]
#A11yMAS
#MAS42A 
#Wipro 
#vscode-website
#Firefox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Prerequisite:**
**_1. Change system setting to High contrast Black_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast Black" from the "Choose a theme" dropdown. Click on Apply.

**Repro Steps:**
1.Open https://code.visualstudio.com/docs/runtimes/nodejs in edge.
2.Navigate to the "Node.js Extensions" above the footer of the page .
3.Observe the download link arrows for the "npm" and "npm Intellisense ","View Node Package " in high contrast black.[screenshot]
Note: Same issue observed for the below link
https://code.visualstudio.com/docs/languages/overview
[screenshot]

**Actual Result:**
The package download link arrows are not visible for the "npm" and "npm Intellisense ","View Node Package " above the footer of the page in high contrast black.

**Expected Result:**
The package download link arrows should be visible for the "npm" and "npm Intellisense ","View Node Package " above the footer of the page in high contrast black.

**User Impact:**
The persons with color blindness and low vision will not be able to download the packages  and may not use the application efficiently.
[Download link arrows not visible.zip](https://github.com/wiproaccess/vscode-website/files/686962/Download.link.arrows.not.visible.zip)

**Comment by waderyan** [comment]
![image](https://cloud.githubusercontent.com/assets/2449568/26735612/7dc4ebc0-4777-11e7-9861-4ede1678a5e8.png)

**Comment by waderyan** [comment]
Fixed with de86107b5b456270965cf5ccc78c4af98d52fd01

**Comment by wiproaccess** [comment]
The issue is verified hence adding the label 'closed' and closing the bug._
